### DISCUSSION TITLE
Mobile Phone Access question

### DISCUSSION THREAD

**Discussion by lazerusrm** [amidaware/tacticalrmm#1741] [General]
is there any option to be able to remotely administer hosts from a mobile device?

I had a splashtop subscription for 15 years and they just cancelled it, now i have to pay 400/year for the same service i was paying 60/year for.

thanks for any and all input!

**Comment by dinger1986** [comment]
You can use meshcentral which is the remotecontrol part of tactical via mobile and it works pretty well, it's accessible via browser.

Tactical itself will work on a mobile but is quite fiddly. Works fine on an iPad sized screen though.

**Reply by lazerusrm** [reply]
So i just use my weblogin at mesh.mydomain.com from a mobile phone? but i have to add all my hosts a second time in that section?

sorry for being dense, im just a little unfamiliar with certain aspects of this.

Thanks again for your response.

**Reply by dinger1986** [reply]
Once you install tactical it will automatically install mesh so the agents will be in both places.

You most likely want to add a new mesh user so you can login directly there.

It uses a login token with tactical so best to connect to an agent then go to mesh.yourdomain.com and create a user with mfa enabled, then you can log in as that user on your mobile.

One word of caution don't remove the tactical mesh group
